integration/qbft/scenarios/regular.go

```diff
@@ -57,115 +61,160 @@ func Regular(role spectypes.BeaconRole) *IntegrationTest {
-			Message: &specqbft.Message{
+				Data:       prepareData,
+			})
+			if !isMessageExistInRound(expectedMsg, actual.State.PrepareContainer.Msgs[specqbft.FirstRound]) {
+				return fmt.Errorf("prepare message %+v wasn't found at actual.State.PrepareContainer.Msgs[specqbft.FirstRound]", expectedMsg)
+			}
+
+			expectedMsg = spectestingutils.SignQBFTMsg(spectestingutils.Testing4SharesSet().Shares[operatorIDIterator], operatorIDIterator, &specqbft.Message{
 				MsgType:    specqbft.CommitMsgType,
 				Height:     specqbft.FirstHeight,
 				Round:      specqbft.FirstRound,
 				Identifier: identifier[:],
-				Data:       spectestingutils.PrepareDataBytes(consensusData),
+				Data:       commitData,
+			})
+			if isMessageExistInRound(expectedMsg, actual.State.CommitContainer.Msgs[specqbft.FirstRound]) {
+				foundedMsgsCounter++
+			}
+		}
+
+		if !actual.State.Share.HasQuorum(foundedMsgsCounter) {
+			return fmt.Errorf("wasn't found enough commit messages at actual.State.CommitContainer.Msgs[specqbft.FirstRound], expected at least %d, actual = %d", actual.State.Share.Quorum, foundedMsgsCounter)
+		}
+
+		actual.State.ProposeContainer = nil
+		actual.State.PrepareContainer = nil
+		actual.State.CommitContainer = nil
+
+		expectedStoredInstance := &protocolstorage.StoredInstance{
+			State: &specqbft.State{
+				Share:             testingShare(spectestingutils.Testing4SharesSet(), operatorID),
+				ID:                identifier[:],
+				Round:             specqbft.FirstRound,
+				Height:            specqbft.FirstHeight,
+				LastPreparedRound: specqbft.FirstRound,
+				LastPreparedValue: consensusData,
+				ProposalAcceptedForCurrentRound: spectestingutils.SignQBFTMsg(spectestingutils.Testing4SharesSet().Shares[1], 1, &specqbft.Message{
+					MsgType:    specqbft.ProposalMsgType,
+					Height:     specqbft.FirstHeight,
+					Round:      specqbft.FirstRound,
+					Identifier: identifier[:],
+					Data:       proposalData,
+				}),
+				Decided:      true,
+				DecidedValue: consensusData,
+
+				RoundChangeContainer: &specqbft.MsgContainer{Msgs: map[specqbft.Round][]*specqbft.SignedMessage{}},
 			},
-		},
+			DecidedMessage: &specqbft.SignedMessage{
+				Message: &specqbft.Message{
+					MsgType:    specqbft.CommitMsgType,
+					Height:     specqbft.FirstHeight,
+					Round:      specqbft.FirstRound,
+					Identifier: identifier[:],
+					Data:       spectestingutils.PrepareDataBytes(consensusData),
+				},
+			},
+		}
+
+		expectedStateRoot, err := expectedStoredInstance.State.GetRoot()
+		if err != nil {
+			return fmt.Errorf("error during geting root from expected state: %w", err)
+		}
+
+		actualStateRoot, err := actual.State.GetRoot()
+		if err != nil {
+			return fmt.Errorf("error during geting root from actual state: %w", err)
+		}
+
+		if !bytes.Equal(expectedStateRoot, actualStateRoot) {
+			return fmt.Errorf("expected and actual state roots doesn't matching, expected state = %+v, actual = %+v", expectedStoredInstance.State, actual.State)
```

Comment:
@egorka-sereda 

- `expected and actual state roots doesn't matching` => `expected and actual state roots don't match`
- if we output both states here, the output will be unreadable because encoded states are very long, I think it's fine to return something like below and put a breakpoint and check states manually if test fails.
```suggestion
			return fmt.Errorf("expected and actual state roots don't match")
```